.github/workflows/release.yml

```diff
@@ -13,6 +13,9 @@ on:
     - 'master'
 env:
   GO111MODULE: on
+permissions:
+  contents: read
```

Comment:
Why no write for making releases?

Reply:
It is only set to `write` for the job that needs it: https://github.com/lima-vm/lima/blob/f0c14f930b62f85468b3f2a21eb0698f73756294/.github/workflows/release.yml#L51-L52